src/styles/index.scss

```diff
@@ -1,3 +1,110 @@
 body {
+  color: rgba(6, 11, 53, 1);
```

Comment:
Consider defining main color values as SCSS variables in a separate file and importing them here. This will make it easier to manage and update colors across your styles.